bin/update-daml-hashes

```diff
@@ -20,10 +20,25 @@ get_hash() (
   out=$(mktemp)
   case $2 in
     public)
+      case "$(uname -m)" in
+          x86_64|amd64)
+              arch="x86_64"
+              ;;
+          aarch64|arm64)
+              if [[ $os == "macos" ]]; then 
```

Comment:
Adding some context to why we need to set the `arch` to `x86_64` when it's on ARM architecture

```suggestion
              # Daml SDK for macOS is only build on x86_64 architecture
              if [[ $os == "macos" ]]; then 
```